operator/v1/types_ingress.go

```diff
@@ -335,8 +335,38 @@ type RouteAdmissionPolicy struct {
 	// If empty, the default is Strict.
 	// +optional
 	NamespaceOwnership NamespaceOwnershipCheck `json:"namespaceOwnership,omitempty"`
+	// wildcardPolicy describes how routes with wildcard policies should
+	// be handled for the ingress controller. WildcardPolicy controls use
+	// of routes [1] exposed by the ingress controller based on the route's
+	// wildcard policy.
+	//
+	// [1] https://github.com/openshift/api/blob/master/route/v1/types.go
+	//
```

Comment:
Please enumerate the possible values here.  (The note below mentions the two possible values, but it does not state that there are no other possible values, and anyway, an explicit enumeration is clearest.)

Reply:
@Miciah good point. I updated the godocs to list the supported values.